packages/phone-number-privacy/signer/docs/deploy-on-aws.md

```diff
@@ -0,0 +1,125 @@
+# Run Phone Number Privacy - Signer Service on AWS
+
+## Prerequisites
+
+- awscli (https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+- AWS credentials configured (https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+
+## Steps
+
+### Setting up the network and database
+
+The next steps will explain how to create a new VPC and a RDS Postgres database needed to run the signer. If desired it is possible to use an existing VPC and a database deployed in some any other way. Take it as a reference for the port/security group configuration.
+
+1. OPTIONAL: [Create new vpc and subnets](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/gsg_create_vpc.html) for the resources (use the CIDR block and AZs that fits with your network setup). The VPC should have at least two subnets, and if it does not have any public subnet (for running the Signer container) the private subnets should have an [Internet Gateway configured](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html)
+
+1. Create the security groups for the database and signer:
+
+    ```bash
+    aws ec2 create-security-group --description "pgpnp database" --group-name pgpnp-db --vpc-id <id>
+    aws ec2 create-security-group --description "signer" --group-name signer --vpc-id <id>
+    ```
+
+1. Create the security group rules The database should expose 5432 (the port could be specified and change during the database creation), and the signer by default uses port 8080:
+
+    ```bash
+    aws ec2 authorize-security-group-ingress --group-id <gpnp-db-id> --protocol tcp --port 5432 --source-group sg-0a1064e7d9cba38a9
```

Comment:
Just confirming, should this be `ec2` since we're using fargate below?

Reply:
Yes, security groups and vpc commands are under `ec2` cli command.